source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/inputhandler/RatingInputHandler.kt

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package io.adaptivecards.renderer.inputhandler
+
+import android.view.accessibility.AccessibilityEvent
+import io.adaptivecards.objectmodel.BaseInputElement
+import io.adaptivecards.renderer.Util
+import io.adaptivecards.renderer.layout.RatingStarInputView
+
+/**
+ * Input handler for Rating input
+ **/
+class RatingInputHandler(
+    baseInputElement: BaseInputElement
+): BaseInputHandler(baseInputElement) {
+    override fun getInput() = (m_view as RatingStarInputView).getRating().toString()
+
+    override fun setInput(input: String?) {
+        // no-op
```

Comment:
This is for inputElement (editable rating)
SetInput should update the UI `(m_view as RatingStarInputView).setRating`

Reply:
Overridden resetValue method required for valueChangedAction. As part of that, provided the implementation of setInput.